### DISCUSSION TITLE
Asset storage in Database

### DISCUSSION THREAD

**Discussion by mickev36** [requarks/wiki#6870] [Help / Questions]
Hello,

I've started using Wiki.js for a personal project, and it seems very promising on all aspects.
However, one point has me concerned : All media assets are stored in Database, not on the filesystem as it traditionally would.
Is there any specific reason to store all media assets in the database instead of the file system ?
Storing large media files in database seem much more costly for system resources, and I'm afraid to encounter issues as the data grows big (uploading numerous large images, etc...).
I've looked through the docs/discussions, but storing assets separately on the filesystem doesn't seem an option as of right now. Am I missing something ?

Thanks in advance

**Comment by NGPixel** [comment]
Assets are stored in the DB as they are meant to be images and documents. You're not supposed to store large file in the wiki itself.

You'll be able to use different storage targets for different file types in v3.

**Comment by mickev36** [comment]
@NGPixel Thanks for your answer.

I understand the wiki's not supposed to store videos and other heavy assets. 
I'm trying to include a couple of screenshots/images for a documentation. They can be numerous and be larger than a small image/icon. Is there a way to reference external images in the wiki before v3 ? I've tried using "Insert media" but it doesn't seem to work, since i'm guessing it's used for embedding videos.

**Reply by AlexJFox** [reply]
You can use the raw html page render and just link to the images externally from another site?

**Reply by AlexJFox** [reply]
Also in markdown you can try `![](https://url.to.image)`
